src/components/course/CoursePage.jsx

```diff
@@ -2,11 +2,17 @@ import React from 'react';
 
 import Course from './Course';
 import AuthenticatedUserSubsidyPage from '../app/AuthenticatedUserSubsidyPage';
+import { SubsidyRequestsContextProvider } from '../enterprise-subsidy-requests';
+import { CourseEnrollmentsContextProvider } from '../dashboard/main-content/course-enrollments';
 
 export default function CoursePage() {
   return (
     <AuthenticatedUserSubsidyPage>
-      <Course />
+      <SubsidyRequestsContextProvider>
```

Comment:
[clarification] Does `SubsidyRequestsContextProvider` need to be added to `CoursePage` when it's already included within `Course`?

Reply:
I didn't notice it included in `Course` - I'll remove it and test again.

Reply:
It's removed and seems to work.